### PR TITLE
VCert SDK - Adds ability to provision certificate using keystore and provider name

### PR DESCRIPTION
**Pull request by luispresuelVenafi**

### CODE REVIEW

**Review by sbanic-venafi — APPROVED** [review]
Are there examples that need to be added/modified, as well?

**Comment by luispresuelVenafi** [comment]
> Are there examples that need to be added/modified, as well?

I did update this one here: https://github.com/Venafi/vcert/pull/469/files#diff-eb556c235da2917708a699331008cd6644dc5ea6baf700c247b35aede5f896a6

I kept the other example as ID, because well, we wanted to have both kind of examples for usage, one for Keystore ID and another for using both Keystore Name and Provider Name

File: internal/datasource/webclient/cloudproviders/genqlient.graphql [hunk 1]
```diff
@@ -10,8 +10,8 @@ mutation ProvisionCertificate(
     }
 }
 
-query GetCloudKeystoresByKeystoreId($cloudKeystoreId: UUID) {
-    cloudKeystores(filter: {cloudKeystoreId: $cloudKeystoreId}) {
+query GetCloudKeystoresByKeystoreInfo($cloudKeystoreId: UUID, $cloudKeystoreName: String, $cloudProviderId: UUID, $cloudProviderName: String) {
```
Comment (rvelaVenafi) [inline internal/datasource/webclient/cloudproviders/genqlient.graphql:13]: can this be called just `GetCloudKeystores`? You are providing all the filters so no need to be specific

**Review by rvelaVenafi — APPROVED** [review]

File: internal/datasource/webclient/cloudproviders/genqlient.graphql (hunk 1 above)
Comment (luispresuelVenafi) [reply internal/datasource/webclient/cloudproviders/genqlient.graphql:13]: No problem

Comment (luispresuelVenafi) [reply internal/datasource/webclient/cloudproviders/genqlient.graphql]: Done by latest changes

**Pull request merged**